packages/app/src/specs/RunStatusDots.cy.tsx

```diff
@@ -28,6 +28,16 @@ function mountWithRuns (runs: Required<CloudSpecRun>[]) {
   })
 }
 
+function mountWithNoData () {
+  cy.mount(() => {
```

Comment:
I will just leave this one as is for now then since it is the same pattern as the other tests.  Is there a "best-practices" guide for how we should write our tests here that gets into bullet points like you are pointing out?  Something like:
> Do not use `beforeEach` or `afterEach` if there is only one test.  Just include the content in the test itself.

Reply:
This guid would probably be the place: https://github.com/cypress-io/cypress/blob/develop/guides/testing-strategy-and-styleguide.md

Though I think there are _some_ valid use cases for the beforeEach with a single test, I don't remember the details.